.github/dependabot.yml

```diff
@@ -10,6 +10,5 @@ updates:
       prefix: "[INFRA]"
       prefix-development: "[INFRA] dev -"
     reviewers:
-      - "aibcmars"
       - "csouchet"
```

Comment:
I suggest we use a team instead of individuals, this will simplify the maintenance
https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#reviewers

We could use: `process-analytics/js-lib-collaborators`

Reply:
I didn't know we could use it. I'll change it 😉

Reply:
Done